### PR TITLE
Output diags to stderr before reconfig

### LINKED ISSUE #9161: Diags are not visible at startup (before reconfigure with records.config)
I noticed that debug messages early in ATS startup are not visible.  It seems that the default config has all log destinations turned off.

### PR DESCRIPTION
**Pull request by cmcfarlen**
fixes #9161

### CODE REVIEW

**Review by bneradt — APPROVED** [review]

**Review by bryancall — APPROVED** [review]

**Comment by ywkaras** [comment]
[approve ci autest]

**Comment by ywkaras** [comment]
[approve ci autest]

**Comment by ywkaras** [comment]
[approve ci autest]

**Comment by cmcfarlen** [comment]
[approve ci]

**Comment by cmcfarlen** [comment]
[approve ci autest]

**Comment by cmcfarlen** [comment]
[approve ci clang-analyzer]

**Comment by cmcfarlen** [comment]
[approve ci fedora]

File: src/tscore/ink_cap.cc [hunk 1]
```diff
@@ -314,14 +314,14 @@ RestrictCapabilities()
     cap_flag_value_t val;
     if (cap_get_flag(caps_good, perm_list[i], CAP_PERMITTED, &val) < 0) {
     } else {
-      Warning("CAP_PERMITTED offiset %d is %s", i, val == CAP_SET ? "set" : "unset");
+      Debug("privileges", "CAP_PERMITTED offiset %d is %s", i, val == CAP_SET ? "set" : "unset");
```
Comment (bneradt) [inline src/tscore/ink_cap.cc:317]: offiset -> offset.

**Review by bneradt — CHANGES_REQUESTED** [review]

**Review by bneradt — APPROVED** [review]
Looks good to me.

**Comment by bneradt** [comment]
[approve ci]

**Pull request merged**